js_modules/dagster-ui/packages/ui-core/src/assets/useAllAssets.tsx

```diff
@@ -22,7 +22,7 @@ export type AssetRecord = Extract<
   {__typename: 'AssetRecordConnection'}
 >['assets'][0];
 
-const POLL_INTERVAL = 60000; // 1 minute
+const POLL_INTERVAL = 60000 * 5; // 5 minutes
```

Comment:
Just calling out that this is a big change

Reply:
I think we really don't need to refetch this much tbh. External assets shouldn't change much and I want to avoid churning the indexeddb cache often (since its slow).